argocd/overlays/moc-infra/applications/envs/moc/smaug/b4mad/racing.yml

```diff
@@ -1,7 +1,7 @@
 apiVersion: argoproj.io/v1alpha1
 kind: Application
 metadata:
-  name: racing
+  name: b4mad-racing
```

Comment:
Note: This will require manual intervention (app resource delete+create) in ArgoCD